src/__tests__/editorManager.spec.js

```diff
@@ -0,0 +1,97 @@
+describe('editorManager', () => {
+  const id = 'testContainer';
+
+  beforeEach(function() {
+    this.$container = $(`<div id="${id}"></div>`).appendTo('body');
+  });
+
+  afterEach(function() {
+    if (this.$container) {
+      destroy();
+      this.$container.remove();
+    }
+  });
+
+  describe('function keys', () => {
+    describe('should begin editing', () => {
+      [
+        'ENTER',
+        'F2',
+      ].forEach((key) => {
+        it(`if ${key} was pressed`, () => {
+          handsontable();
+
+          selectCell(0, 0);
+
+          const activeEditor = getActiveEditor();
+
+          keyDownUp(key.toLowerCase());
+
+          expect(activeEditor.isOpened()).toBe(true);
+        });
+      });
+    });
+
+    describe('should not begin editing', () => {
+      [
+        'ALT',
```

Comment:
I wonder if we should have a division into <kbd>Alt</kbd> and <kbd>AltGr</kbd> for better management of it in the future. https://en.wikipedia.org/wiki/AltGr_key. Each key works differently in a Handsontable.
<kbd>AltGr</kbd> (right alt key) with selection enables multiselection, left alt not.

Reply:
<kbd>AltGr</kdb> under the hood uses <kbd>Ctrl</kdb>+<kbd>Alt</kdb>. We support selecting non-contiguous cells if user press <kbd>Ctrl</kbd> key.

From the `event.keyCode` (the property we use to recognize what key is used), we get `18` for both, `Alt` and `AltGraph`.

Reply:
> So I suggest moving that point to the separated issue due to we do not support `AltGr` anywhere in Handsontable.

Good point.